src/commands/createHttpFunction.ts

```diff
@@ -65,3 +62,29 @@ function generateSuffixedName(preferredName: string, i: number): string {
 async function isNameAvailable(folderPath: string, newName: string): Promise<boolean> {
     return !(await fse.pathExists(path.join(folderPath, newName)));
 }
+
+async function getFunctionsApi(context: IActionContext): Promise<AzureFunctionsExtensionApi> {
+    const funcExtensionId: string = 'ms-azuretools.vscode-azurefunctions';
+    const funcExtension: vscode.Extension<AzureExtensionApiProvider> | undefined = vscode.extensions.getExtension(funcExtensionId);
+
+    if (funcExtension) {
+        if (!funcExtension.isActive) {
+            await funcExtension.activate();
+        }
+
+        // The Functions DB extension just recently added support for 'AzureExtensionApiProvider' so we should do an additional check just to makes sure it's defined
```

Comment:
This is not true - it was added a year and a half ago: https://github.com/microsoft/vscode-azurefunctions/pull/812. You don't need the extra check for `funcExtension.exports`